src/sentry/options/defaults.py

```diff
@@ -1470,36 +1470,6 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
-# Enables extrapolation on the `transactions` namespace.
-register(
```

Comment:
Either remove these in a separate PR, or ensure that https://github.com/getsentry/sentry-options-automator/pull/2199 is merged _and applied_ before this PR.